plugin/evm/gossip.go

```diff
@@ -84,8 +85,8 @@ func (t txGossipHandler) CrossChainAppRequest(context.Context, ids.ID, time.Time
 	return nil, nil
 }
 
-func NewGossipEthTxPool(mempool *txpool.TxPool) (*GossipEthTxPool, error) {
-	bloom, err := gossip.NewBloomFilter(txGossipBloomMinTargetElements, txGossipBloomTargetFalsePositiveRate, txGossipBloomResetFalsePositiveRate)
+func NewGossipEthTxPool(mempool *txpool.TxPool, registerer prometheus.Registerer) (*GossipEthTxPool, error) {
+	bloom, err := gossip.NewBloomFilter(registerer, "eth_tx_bloom_filter", txGossipBloomMinTargetElements, txGossipBloomTargetFalsePositiveRate, txGossipBloomResetFalsePositiveRate)
```

Comment:
Do we actually want the `eth` prefix here? It's only done in coreth because of the different transaction types.
```suggestion
	bloom, err := gossip.NewBloomFilter(registerer, "tx_bloom_filter", txGossipBloomMinTargetElements, txGossipBloomTargetFalsePositiveRate, txGossipBloomResetFalsePositiveRate)
```
If this was done to stay consistent between coreth and subnet-evm, feel free to ignore.